docs/source/configuration/persisted-queries.mdx

```diff
@@ -40,52 +38,54 @@ For more information on other configuration aspects, see the [GraphOS persisted
 
 The router provides four configuration options that you can combine to create the recommended [security levels](#router-security-levels). This section details each configuration option. Refer to the [security levels](#router-security-levels) section for recommended combinations.
 
-#### `preview_persisted_queries`
+> From version `1.25.0` to `1.xx.0` the `persisted_queries` configuration option was called `preview_persisted_queries`. Upgrade your router to version `1.xx.0` or later to use the [generally available](/resources/product-launch-stages/#general-availability) version of the feature and the example configuration snippets below.
```

Comment:
@abernix , would you let me know what router version this ends up being released in please?